backend/lcfs/web/api/compliance_report/repo.py

```diff
@@ -142,7 +139,7 @@ async def get_compliance_period(self, period: str) -> CompliancePeriod:
         return result
 
     @repo_handler
-    async def get_compliance_report(
+    async def check_compliance_report(
```

Comment:
question: Why rename this to check_compliance_report when it still returns a compliance report?

Reply:
unlike in get_compliance_report_id, where it joins and returns all related data, this method is only used to check the existence of the compliance report id in the main table. And since it's used in only validations, so had it renamed.